dotnet-client-libraries/HowToUseSftpConnectionForFileUploadInDotNet/Test/WinScpLiveTests.cs

```diff
@@ -0,0 +1,58 @@
+﻿using WinSCP;
+
+namespace Test;
+
+/**
+ * We can easily set a test server to run these tests against using "Rebex Tiny SFTP Server"
+ * 
+ * https://www.rebex.net/tiny-sftp-server/
+ */
```

Comment:
Altough the approach in the link is straightforward, do you think it's possible to run an SFTP server as a SETUP step of the code, instead of requesting a manual set up, something like described here : https://blog.rebex.net/write-your-own-sftp-server-in-csharp

Let me know if that's possible, otherwise we can keep it as is. 
Thanks!

Reply:
I did a quick test and seems that it is not possible to run a console application with the SFTP server and use the unit test runner at the same time using only the Visual Studio UI. 

Users would need to run the server with VS and, for example, run the tests using the command line runner. Seems to me that it wouldn't necessarily make the user experience better

What do you think?